src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectwizard/viewmodel/ProjectWizardViewModel.kt

```diff
@@ -52,6 +55,17 @@ class ProjectWizardViewModel : ViewModel() {
         }
     }
 
+    private fun filterToAvailableLanguges() {
```

Comment:
Although `filterToAvailableLanguges` is misspelled, it's nice and descriptive.  Consider renaming the `filteredLanguages` field to `availableLanguages` so they match.